crates/bevy_gltf/src/lib.rs

```diff
@@ -111,3 +114,19 @@ pub struct GltfPrimitive {
 pub struct GltfExtras {
     pub value: String,
 }
+
+/// Names of [morph targets] for this mesh.
+///
+/// [morph targets]: bevy_render::mesh::morph::MorphWeights
+#[derive(Deserialize, Clone, Debug, Reflect, Default, Component)]
+#[serde(rename_all = "camelCase")]
+#[reflect(Component)]
+pub struct MorphTargetNames {
+    /// Names of morph targets in the [`MorphWeights`] component.
+    ///
+    /// The the name at index `i` in this `Vec` corresponds to the weight
```

Comment:
```suggestion
    /// The name at index `i` in this `Vec` corresponds to the weight
```

Reply:
Hopefully I don't have to rewrite all of the loader bit after asset preprocessing lands :sweat_smile: